chain/chain/src/crypto_hash_timer.rs

```diff
@@ -0,0 +1,72 @@
+use lru::LruCache;
+use near_primitives::{hash::CryptoHash, time::Clock};
+use std::{
+    sync::Mutex,
+    time::{Duration, Instant},
+};
+
+use once_cell::sync::Lazy;
+
+// Cache with the mapping from CryptoHash (blocks, chunks) to the number milliseconds that it took to process them.
+// Used only for debugging purposes.
+pub static CRYPTO_HASH_TIMER_RESULTS: Lazy<Mutex<LruCache<CryptoHash, Duration>>> =
```

Comment:
```suggestion
static CRYPTO_HASH_TIMER_RESULTS: Lazy<Mutex<LruCache<CryptoHash, Duration>>> =
```

I thing should be enough. If not, that's suspect -- publicly visible global state is especially not great